src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -637,6 +638,7 @@
         "Power Topology": ["FeatureMap"],
         "Valve Configuration and Control": ["RemainingDuration"],
         "Boolean State Configuration": ["CurrentSensitivityLevel"],
+        "Water Heater Management": ["FeatureMap"],
```

Comment:
This should be added with the implementation, no?  Right now there is no AttributeAccessInterface at all.

Reply:
Would you like to see this line removed until the implementation is submitted for review?

Reply:
Yes, please.  At that point, the line can be checked against the actual implementation to make sure they match.

Reply:
Line removed